File: api/routes/comments.js
let express = require('express');
let router = express.Router();
let passport = require('passport');

let db = require('../queries/comments');

router.get('/comments', passport.authenticate('jwt', { session: false }), db.getComments);

router.post('/comments', passport.authenticate('jwt', { session: false }), db.createComment);

router.put('/comments/:id', passport.authenticate('jwt', { session: false }), db.updateComment);

router.delete('/comments/:id', passport.authenticate('jwt', { session: false }), db.deleteComment);

module.exports = router;

File: go-api/routes/comments.go
package routes

import (
	"../queries"
)

func initComments() {
	Router.DELETE("/comments/:id", queries.DeleteComment)
	Router.GET("/comments", queries.GetComments)
	Router.POST("/comments", queries.CreateComment)
	Router.GET("/comments/:id", queries.GetComment)
	Router.PUT("/comments/:id", queries.UpdateComment)
}